CODEOWNERS

```diff
@@ -278,8 +278,7 @@
 /scripts/elf_helper.py                    @andrewboie
 /scripts/sanity_chk/expr_parser.py        @nashif
 /scripts/gen_app_partitions.py            @andrewboie
```

Comment:
> which contains only the image names, instead of the '|'-separated struct. 

Create z which contains x, instead of y.

It only makes sense to say "instead of y", if you were changing z, not creating it. The language needs work here.

Reply:
This is also something that is happening partially in fw-nrfconnect-nrf.

Reply:
I updated the commit message a bit, and added a define_property for `PM_IMAGES`.